includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -113,10 +113,20 @@ public function register_routes() {
 						'type'              => 'boolean',
 						'validate_callback' => 'rest_validate_request_arg',
 					],
+					'is_short_statement_descriptor_enabled' => [
+						'description'       => __( 'When enabled, we\'ll include the order number for card and express checkout transactions.', 'woocommerce-payments' ),
+						'type'              => 'boolean',
+						'validate_callback' => 'rest_validate_request_arg',
+					],
+					'short_statement_descriptor'        => [
+						'description'       => __( 'We\'ll use the short version in combination with the customer order number.', 'woocommerce-payments' ),
+						'type'              => 'string',
+						'validate_callback' => [ $this, 'validate_short_statement_descriptor' ],
+					],
 					'account_statement_descriptor'      => [
 						'description'       => __( 'WooCommerce Payments bank account descriptor to be displayed in customers\' bank accounts.', 'woocommerce-payments' ),
 						'type'              => 'string',
-						'validate_callback' => [ $this, 'validate_statement_descriptor' ],
+						'validate_callback' => [ $this, 'validate_regular_statement_descriptor' ],
```

Comment:
Nit: Maybe we should call this `full` instead of `regular`, to not introduce a new term.

Reply:
Renamed it in 389e24b.